src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1407,21 +1407,21 @@ void Graph::DropNode(const NodePtr &node) {
         auto parent = p_edge->getParent();
         if (!parent) continue;
 
+        EdgePtr remEdge = p_edge;
+        int inNum = 0;
+        if (remEdge) {
```

Comment:
Minor. Variable `remEdge` and this check are not needed as well. `p_edge` already validated on line L1406. The same for `c_edge`.

Reply:
Yes indeed. `remEdge` and unnecessary check were removed, thank you.